tools/ci_build/github/linux/docker/scripts/install_deps.sh

```diff
@@ -37,7 +37,7 @@ rm -rf /tmp/src
 DISTRIBUTOR=$(lsb_release -i -s)
 if [ "$DISTRIBUTOR" = "Ubuntu" ]; then
   apt-get -y remove libprotobuf-dev protobuf-compiler
-else
+elif [ "$DISTRIBUTOR" != "CentOS" ]; then
```

Comment:
We need to uninstall  protobuf-devel protobuf-compiler if they are installed.  Here this 'elif' looks confusing.

Reply:
Why should it be uninstalled ? There's no comment about this and the build is working fine so I did not bother to uninstall only the dev/compiler parts of a full installation from sources.

Reply:
onnxruntime  depends on protobuf 3.x.
However, the protobuf lib in ubuntu repo, is 2.x.

If we have both of them on the disk, onnxruntime may wrongly pick up the 2.x version header file

Reply:
Here I skip uninstalling on CentOS / manylinux because we never installed protobuf system package in the first place but used install_protobuf.sh which installs protobuff 3.6.1, we should be safe here.

Reply:
Please add a comment there. Because install_deps.sh  doesn't know why CentOS doesn't have this package installed. 
And, what ever how did you installed protobuf, it's safer to uninstall it before building onnxruntime. Because, it's lesser chance to have version conflict.

Reply:
It's already changed in the latest commit, I forgot to give you a heads-up. Please check the new diff on this file.